homeassistant/components/iometer/quality_scale.yaml

```diff
@@ -0,0 +1,60 @@
+rules:
+  # Bronze
+  action-setup: done # does not apply
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
```

Comment:
I think that the [last point of this rule](https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/dependency-transparency/) is not met yet:
> The version of the dependency published on PyPI should correspond to a tagged release in an open online repository.

Reply:
The dependency is under the MIT license and it is available on PyPi [here](https://pypi.org/project/iometer/). The package is published to PyPi with a github action that has to be triggered manually, as on push to main is sometimes not wanted. When providing new documentation for instance. Is this a concern? Also I saw that a tagged release is missing. I am going to fix this.

Reply:
Done

Reply:
It seems that the tag was created on the wrong commit. 
The version on pypi comes from commit https://github.com/iometer-gmbh/iometer.py/commit/c72118906d42fcc412a4fab1d493ced4927a23e0 while the tag is on https://github.com/iometer-gmbh/iometer.py/commit/a7155627ade168983ad315218f75bd0d61f1f26d

Can you delete the tag and recreate it on the correct commit please?

Reply:
Oh my bad, done.

Reply:
thanks!